### PR TITLE
Add option for the release-script

### PR DESCRIPTION
**Pull request by UncleSamSwiss**
**PR Checklist:**

-   [x] Provide a meaningful description to this PR or mention which issues this fixes.
-   [x] Add tests for your change. This includes negative tests (i.e. inputs that need to fail) as well as baseline tests (i.e. how should the directory structure look like?).
-   [x] Run the test suite with `npm test`
-   [x] If there are baseline changes, review them and make a separate commit for them with the comment "accept baselines" if they are desired changes
-   [x] Ensure the project builds with `npm run build`
-   [x] If you added a required option, also add it to the template creation (`.github/create_templates.ts`)
-   [x] Add a detailed migration description to `docs/updates` explaining what the user needs to do when manually updating an existing project
-   [x] Add your changes to `CHANGELOG.md` (referencing the migration description and this PR or the issue you fixed)

**Description:**  
The option is set to "yes" by default as we want to push people to use the release-script as much as possible.

### CODE REVIEW

File: src/lib/core/questions.ts [hunk 1]
```diff
@@ -642,6 +642,19 @@ export const questionGroups: QuestionGroup[] = [
 							: null,
 					].filter((f) => !!f) as string[],
 			}),
+			{
+				type: "select",
+				name: "releaseScript",
+				label: "Release Script",
+				message:
+					"Would you like to create new releases directly from the command line?",
```
Comment (AlCalzone) [inline src/lib/core/questions.ts]: Maybe "sell" this option a bit more? What do you think?
```suggestion
					"Would you like to automate new releases with one simple command?",
```

File: templates/README.md.ts [hunk 2]
```diff
@@ -136,7 +140,15 @@ ${useGithubActions ? `Since you have chosen GitHub Actions as your CI service, y
...
 \`v<major>.<minor>.<patch>\`. The necessary steps are described in \`.github/workflows/test-and-release.yml\`.
 
-`: ""}To get your ${isAdapter ? "adapter" : "widget"} released in ioBroker, please refer to the documentation 
+`: ""}${useReleaseScript ? `Since you installed the release script, you can create a new
+release simply by calling:
+\`\`\`bash
+npm run release
+\`\`\`
+Additional command line options for the release script are explained in 
+[AlCalzone/release-script](https://github.com/AlCalzone/release-script#command-line).
```
Comment (AlCalzone) [inline templates/README.md.ts]:
```suggestion
Additional command line options for the release script are explained in the
[release-script documentation](https://github.com/AlCalzone/release-script#command-line).
```

File: templates/README.md.ts [hunk 3]
```diff
@@ -53,6 +54,9 @@ export = (answers => {
 	if (useESLint) {
 		npmScripts["lint"] = "Runs \`ESLint\` to check your code for formatting errors and potential bugs.";
 	}
+	if (useReleaseScript) {
+		npmScripts["release"] = "Creates a new release, see [@alcalzone/release-script](https://github.com/AlCalzone/release-script#usage) for more details.";
```
Comment (AlCalzone) [inline templates/README.md.ts]:
```suggestion
		npmScripts["release"] = "Creates a new release, see [`@alcalzone/release-script`](https://github.com/AlCalzone/release-script#usage) for more details.";
```

Comment (AlCalzone) [reply templates/README.md.ts]: Or should we also call this "release-script documentation" like below?

File: test/baselines/releaseScript/README.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+1. Upload the resulting file to your ioBroker host
+1. Install it locally (The paths are different on Windows):
+	```bash
+	cd /opt/iobroker
+	npm i /path/to/tarball.tgz
+	```
+
+For later updates, the above procedure is not necessary. Just do the following:
+1. Overwrite the changed files in the adapter directory (`/opt/iobroker/node_modules/iobroker.test-adapter`)
+1. Execute `iobroker upload test-adapter` on the ioBroker host
```
Comment (AlCalzone) [inline test/baselines/releaseScript/README.md:106]: By the way - we should replace this with the dev-server in a separate PR.
Or at least mention it as the preferred alternative and keep this as the "fallback" manual solution.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Just a few nits

File: test/baselines/releaseScript/README.md (hunk 4 above)
Comment (UncleSamSwiss) [reply test/baselines/releaseScript/README.md:106]: wilco

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**